src/Integration/State/StateManager.cs

```diff
@@ -39,6 +40,7 @@ public StateManager(IHost host, TransferableVisualState state)
 
             this.Host = host;
             this.ManagedState = state;
+            this.isSolutionBound = this.ManagedState.HasBoundProject;
```

Comment:
@eyalantzMS When I first open a solution which is already bound to a SQ project, this `this.ManagedState.HasBoundProject` returns `false` until the SonarQube pane is opened in the Team Explorer. It's probably because we want to delay the initialization. Is there a way I could set this flag to the correct value?